compiler/qsc_rca/src/errors.rs

```diff
@@ -161,6 +161,11 @@ pub enum Error {
     #[diagnostic(code("Qsc.CapabilitiesCk.MeasurementWithinDynamicScope"))]
     MeasurementWithinDynamicScope(#[label] Span),
 
+    #[error("cannot define a custom measurement")]
+    #[diagnostic(help("cannot define a custom measurement in the configured target profile"))]
+    #[diagnostic(code("Qsc.CapabilitiesCk.CustomMeasurement"))]
```

Comment:
We'll want to add a section on the wiki page for this error with some explanation and update this error to include the link. We can take a look at what that section should say and get that updated and link added here before the PR goes in so that we don't forget to follow up!